modules/backend/package.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "@wintercms/wn-backend-module",
+    "version": "1.0.0",
+    "description": "The Backend module for Winter CMS.",
+    "private": true,
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/wintercms/winter.git"
+    },
+    "author": {
+        "name": "Ben Thomson",
+        "email": "git@alfreido.com",
+        "url": "https://wintercms.com"
+    },
+    "contributors": [
+        {
+            "name": "Winter CMS Maintainers",
+            "url": "https://wintercms.com"
+        }
+    ],
+    "license": "MIT",
+    "bugs": {
+        "url": "https://github.com/wintercms/winter/issues"
+    },
+    "homepage": "https://wintercms.com/",
+    "dependencies": {
+        "@popperjs/core": "^2.11.4",
```

Comment:
What is this used for?

Reply:
It was part of the Inspector rewrite. It's not used anywhere at the moment, so I'll remove it.